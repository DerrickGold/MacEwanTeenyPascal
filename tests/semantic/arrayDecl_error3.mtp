(* Declare an array with variable index *)


var index : integer;
begin

	var something : array(index) of integer;
	begin
	end

end.