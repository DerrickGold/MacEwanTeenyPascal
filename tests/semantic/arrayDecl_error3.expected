
Semantic Error [line 7]: 
	Invalid array declaration: 
	Expected 'Constant Integer' index type; found type: 'Variable'
